java-lotto/src/main/java/domain/Lotto/LottoService.java

```diff
@@ -3,8 +3,17 @@
 import java.util.*;
 
 public class LottoService {
-    private static final int LOTTO_NUMBER_COUNT = 6;
-    private static final int LOTTO_MAX_NUMBER = 45;
+    private static final int LOTTO_PRICE = 1000;
+
+    public Lottos generateLottos(LottoRequest request, int purchaseAmount) {
+        int manualCount = request.getManualTicketCount();
+        int autoCount = (purchaseAmount / LOTTO_PRICE) - manualCount;
```

Comment:
LottoRequest 객체를 활용해 봐도 좋을 것 같습니다

```suggestion
        int autoCount = request.autoCount();
```